tests/testthat/test-object-package.R

```diff
@@ -17,3 +17,94 @@ test_that("person turned into meaningful text", {
     person_desc(comment = c("ORCID" = "1234", "extra"))
   })
 })
+
+test_that("package description not affected if no links", {
+  text <- "A simple description with no links."
+
+  parsed <- package_description_urls(text)
+
+  expect_equal(
+    parsed,
+    text
+  )
+  expect_snapshot(parsed)
```

Comment:
There's no need to include a snapshot when you've already tested the results with `expect_equal()`